flumeDevice

```diff
@@ -444,21 +444,24 @@ def processQueryResponse(resp)
             }
 
             runTime = Math.round((currentFlowTS - flowDetectedTS)/600)/100
-            sendEvent(name: "flowDurationDetail", value: status + " for " + runTime.toString() + " min")
+            sendEvent(name: "flowDetail", value: "flow detected")
             sendEvent(name: "flowDurationMin", value: runTime)
```

Comment:
I think this wording is what we are circling on now ...

- ""
- "flow detected" (this works well with L455 `"waiting for first flow detection"`
- "analyzing water usage"
- ???

Reply:
I went with `flow detected` .. I think this is works, and doesnt introduce as much confusion/?

Reply:
ripped it out .. no loss